### PR TITLE
updated content publisher notebooks

### PR DESCRIPTION
**Pull request by shivanip32**
These are updated content publisher notebooks

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [x] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [x] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('https://www.arcgis.com', 'arcgis_python', 'P@ssword123')`
    - `gis = GIS(profile="your_online_profile")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal', 'arcgis_python', 'amazing_arcgis_123')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [x] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [x] Code refactored & split out across multiple cells, useful comments?
- [x] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [x] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [x] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [x] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

**Comment by shivanip32** [comment]
https://github.com/ArcGIS/geosaurus/issues/4685

File: samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb [hunk 1]
```diff
@@ -38,7 +38,7 @@
    "metadata": {},
```
Comment (AtmaMani) [inline samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb:38]: <p>Please use a profile or 'home' mode</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: samples/05_content_publishers/pdf_table_to_pdf_map.ipynb [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "\n",
```
Comment (AtmaMani) [inline samples/05_content_publishers/pdf_table_to_pdf_map.ipynb:34]: <p>Can we switch back to using a profile?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Comment by AtmaMani** [comment]
@DavidJVitale could you review the sample that identifies insecure URLs?
@CMPeng could you review the sample that extracts content from pdf and publishes as items?

**Comment by CMPeng** [comment]
@AtmaMani I have finished reviewing the `pdf_table_to_pdf_map.ipynb` sample - the changes look good to me, and thanks @shivanip32 for updating the results and specifying the version number of library. I also found that the u/p is exposed and the profile param is not switched back.

**Comment by priyankatuteja** [comment]
@shivanip32 Please resolve merge conflicts

File: samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb (hunk 1 above)
Comment (jyaistMap) [reply samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb:38]: <p>I pushed a pull request to @shivanip32's branch changing this to</p><p><em>gis = GIS(profile="your_online_profile")</em></p>

Comment (jyaistMap) [reply samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb:38]: <p>I pushed a second pull request changing it to</p><p><em>gis = GIS('home')</em></p>

**Comment by shivanip32** [comment]
Thanks @jyaistMap for pointing out the u/p issue. I have updated GIS credentials with GIS('Home') for arcgis online account.

**Review by jyaistMap — APPROVED** [review]
lgtm

File: samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb (hunk 1 above)
Comment (priyankatuteja) [inline samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb:38]: <p>workspace = "/arcgis/home" </p><p> </p><p>@shivanip32 Should the workspace path be data/arcgis/home?</p><p>To make this sample running on try it live. I think we can upload a <strong>csv collection</strong> item and use it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Review by priyankatuteja — CHANGES_REQUESTED** [review]
@shivanip32 You need to remove the output csv files, only input ones can be added to the data folder as discussed

**Comment by shivanip32** [comment]
@priyankatuteja, I have deleted the output csv files from the folder.

**Review by priyankatuteja — APPROVED** [review]

**Comment by AtmaMani** [comment]
Thank you @shivanip32 @priyankatuteja @jyaistMap @CMPeng

File: samples/05_content_publishers/pdf_table_to_pdf_map.ipynb (hunk 2 above)
Comment (jyaistMap) [reply samples/05_content_publishers/pdf_table_to_pdf_map.ipynb:34]: This now uses _gis = GIS('home')_

File: samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb (hunk 1 above)
Comment (jyaistMap) [reply samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb:38]: @shivanip32 Can you verify whether the `workspace` is correct and resolve this conversation if it is?

Comment (jyaistMap) [inline samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb:38]: <ul><li>'Home' should be changed to <em>'home'</em></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (jyaistMap) [inline samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb:38]: <p>Why not use Path to verify/make the directory?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Comment by jyaistMap** [comment]
Made a final pass:
 * suggested typo fix to `gis = GIS('Home')`
 * noted a couple places where credentials still are in the notebooks
 * noted in one cell in the insecure/secure notebook where both `os.path` and `pathlib.Path` are used - can be changed for consistency, but not necessary
 * asked for resolution on note about whether `workspace` path is correct

**Pull request merged**